### PR TITLE
Tests + Fix for new BB merge handling

### PR DESCRIPTION
**Pull request by gardnerjack**

### CODE REVIEW

**Comment by gardnerjack** [comment]
**Dev deployment:** https://bitbucket.org/atlassian/atlassian-frontend-landkid-deployment/addon/pipelines/home#!/results/259

File: tests/integration/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../../tsconfig.base.json",
+    "compilerOptions": {
+        "types": ["cypress"]
+    }
+}
```
Comment (RajaBellebon) [inline tests/integration/tsconfig.json:6]: I think it needs a `\n`

File: tsconfig.json [hunk 2]
```diff
@@ -1,25 +1,7 @@
...
-        "noImplicitAny": true
+        "outDir": "dist"
     },
-    "exclude": [
-        "node_modules",
-        "public"
-    ]
-}
+    "exclude": ["node_modules", "tests"]
+}
```
Comment (RajaBellebon) [inline tsconfig.json:7]: same

File: jest.config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testMatch: ['**/tests/unit/*.test.ts'],
```
Comment (RajaBellebon) [inline jest.config.js:4]: just my preference should not really matter but I do `__tests__/unit/**`

**Pull request merged**